Replicator/Replicator.cc

```diff
@@ -30,6 +30,8 @@
 #include "Instrumentation.hh"
 #include "fleece/Mutable.hh"
 
+#include <iostream>
```

Comment:
Is this include required?

Reply:
You're right, it isn't. I think it got auto-included by the IDE when I was trying to output something.